src/main/java/club/sk1er/mods/core/universal/UGraphics.java

```diff
@@ -442,6 +445,31 @@ public static DynamicTexture getEmptyTexture() {
         //#endif
     }
     //#endif
+    //#if FABRIC
+    //$$ public static NativeImageBackedTexture getTexture(InputStream stream) {
```

Comment:
Does `NativeImageBackedTexture` have the same API as `DynamicTexture`? If not, it might be worth making a small `UTexture` to wrap both of these classes.

Reply:
``NativeImageBackedTexture`` and ``DynamicTexture`` have the same API except for the primary constructor taking a ``NativeImage`` instead of a ``BufferedImage`` on newer versions (1.15 + 1.16).

![image](https://user-images.githubusercontent.com/71222289/112876385-5f7a4780-90bd-11eb-8d11-81cadf42e3b5.png)

It would be possible to make a ``UTexture`` class that wraps these classes from the looks of it. I'll add that now if you wish! :smile:

Reply:
We can do this later if we need to